packages/nimble-components/src/rich-text-editor/styles.ts

```diff
@@ -1,12 +1,159 @@
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    borderHoverColor,
+    borderRgbPartialColor,
+    borderWidth,
+    smallDelay,
+    standardPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        inline-size: auto;
+        block-size: 100%;
+        flex-direction: column;
+        position: relative;
+        --ni-private-hover-indicator-width: calc(${borderWidth} + 1px);
+        ${
+            /**
+             * Min inline size to accommodate all the possible buttons in the toolbar and max inline size is referred from the visual design.
+             */ ''
+        }
+        min-inline-size: 360px;
+        max-inline-size: 720px;
+    }
+
+    .container {
+        display: flex;
+        flex-direction: column;
+        block-size: inherit;
+        position: relative;
+        max-block-size: 100%;
+        border: ${borderWidth} solid rgba(${borderRgbPartialColor}, 0.3);
+    }
+
+    .container::after {
+        display: block;
+        content: ' ';
+        position: absolute;
+        bottom: calc(-1 * ${borderWidth});
+        inline-size: 0px;
+        block-size: 0px;
+        bottom: 0%;
+        left: 50%;
+        transform: translate(-50%, 50%);
+        border-bottom: ${borderHoverColor}
+            var(--ni-private-hover-indicator-width) solid;
+        transition: width ${smallDelay} ease-in;
+    }
+
+    .container:focus-within {
+        border-bottom-color: ${borderHoverColor};
+    }
+
+    @media (prefers-reduced-motion) {
+        .container::after {
+            transition-duration: 0s;
+        }
+    }
+
+    :host(:hover) .container::after {
+        inline-size: 100%;
+    }
+
+    .editor {
+        border: ${borderWidth} solid transparent;
+        border-radius: 0px;
+        block-size: calc(100% - 44px);
+        overflow: auto;
+    }
+
+    .ProseMirror {
+        ${
+            /**
+             * Min block size represents the one line space for the initial view and max block size is referred from the visual design.
+             * However, max block size will be `fit-content` when the `fit-to-content` attribute is implemented.
+             */ ''
+        }
+        min-block-size: 32px;
+        max-block-size: 132px;
+        block-size: 100%;
+        border: ${borderWidth} solid transparent;
+        border-radius: 0px;
+        background-color: transparent;
+        font: inherit;
+        padding: 8px;
+        box-sizing: border-box;
+        position: relative;
+        color: inherit;
+        ${
+            /**
+             * Below are the styles from prosemirror-view.
+             */ ''
+        }
+        word-wrap: break-word;
+        white-space: pre-wrap;
```

Comment:
Overriden by next line's `white-space`? I also think that `pre-wrap` will preserve sequence of white space characters and gives opportunity for line breaks only at end of that sequence whereas `break-spaces` gives opportunity to add a line break after each white space character, resulting in next line having the left out white space. 

Is `pre-wrap` more preferrable over `break-spaces`? If yes, consider removing the next line.

Reply:
If this is a default class coming from ProseMirror, please point to the documentation (if any) for rationale of this class.

Reply:
I have included the link to the styles within the file. If the white-space property is omitted, a console warning message prompts the addition of styles from `prosemirror-view`. This is the reason for incorporating this particular style in this class.

![image](https://github.com/ni/nimble/assets/123377523/f6f9a6c6-911f-4e59-926d-90648574a900)